fsharp-backend/src/LibExecution/RuntimeTypes.fs

```diff
@@ -169,25 +170,38 @@ module DDateTime =
   let toIsoString (d : T) : string = (toInstant d).toIsoString ()
 
 
-// This Expr is the AST, expressing what the user sees in their editor.
+/// The AST, expressing what the user sees in their editor.
 type Expr =
   | EInteger of id * int64
   | EBool of id * bool
   | EString of id * string
   | ECharacter of id * string
-  | EFloat of id * double // first string might have a sign in it
+  | EFloat of id * double // first string might have a sign in it <- what?
   | ENull of id
   | EBlank of id
+
+  /// Composed of binding name, expression, and ???
```

Comment:
what's the 2 expressions here? I assume one of them is _the_ expression being bound, but not sure what the other is.

Reply:
```
let str = expr1
expr2
```